src/Evolution/Systems/GrMhd/GhValenciaDivClean/System.hpp

```diff
@@ -47,13 +47,24 @@ struct System {
   using gradient_variables =
       tmpl::append<typename gh_system::gradient_variables,
                    typename grmhd_system::gradient_variables>;
-  using gradient_tags = gradient_variables;
+  using gradients_tags = gradient_variables;
   using sourced_variables = typename grmhd_system::sourced_variables;
+  // This causes some extra derivatives of GRMHD quantities to be calculated
```

Comment:
could you say where the derivatives are computed? I thought during the evolution only derivatives of the `gradient_variables` are computed, so why are the GRMHD variables listed there? Though I assume the `grmhd_system::gradient_variables` is empty, so now I'm really confused what this comment is telling me :P

Reply:
Ah, yep, I had misunderstood the `DerivCompute` tag added in `TimeStepperHistory` -- I think you're right that there's no redundancy because the `grmhd_system::gradient_variables` is appropriately an empty list